packages/react-ui-core/src/Dropdown/Dropdown.js

```diff
@@ -53,6 +53,16 @@ export default class Dropdown extends Component {
     return parseArgs(anchorField, AnchorButton)
   }
 
+  renderChildren() {
+    const props = { toggleVisibilty: this.toggleVisibilty }
+    const children = React.Children.toArray(this.props.children)
+    /* eslint-disable no-confusing-arrow */
```

Comment:
I'd rather not do this if we don't have to. Can't we just add the parens to the following line instead?

Reply:
👍